rollup/internal/orm/chunk.go

```diff
@@ -259,6 +278,49 @@ func (o *Chunk) InsertChunk(ctx context.Context, chunk *encoding.Chunk, codecVer
 	return &newChunk, nil
 }
 
+// InsertTestChunkForProposerTool inserts a new chunk into the database only for testing purposes in the proposer tool.
```

Comment:
why can't reuse the `InsertChunk`

Reply:
This function hacks some vars, e.g., `chunk index` and `queue index`, so it's better to use another test-only function to do it.